icons/src/symlinks/symbolic/categories.list

```diff
@@ -3,7 +3,6 @@ applications-utilities-symbolic.svg applications-accessories-symbolic.svg
 calendar-today-symbolic.svg view-tasks-today-symbolic.svg
 calendar-week-symbolic.svg view-tasks-week-symbolic.svg
 diagnostics-symbolic.svg org.gnome.Settings-diagnostics-symbolic.svg
-multitasking-symbolic.svg org.gnome.Settings-multitasking-symbolic.svg
 multitasking-symbolic.svg preferences-desktop-multitasking-symbolic.svg
```

Comment:
Shouldn't we keep `org.gnome.Settings-multitasking-symbolic.svg` instead?

Reply:
In fact this article is already [here](https://github.com/ubuntu/yaru/blob/master/icons/src/symlinks/symbolic/apps.list#L97)， so it is still duplicated